src/csharp/Grpc.Core/AsyncDuplexStreamingCall.cs

```diff
@@ -117,6 +117,10 @@ public Metadata GetTrailers()
         /// Otherwise, requests cancellation of the call which should terminate all pending async operations associated with the call.
         /// As a result, all resources being used by the call should be released eventually.
         /// </summary>
+        /// <remarks>
+        /// Normally, there is no need for you to dispose the call unless you want to utilize the
+        /// "Cancel" semantics of invoking <c>Dispose</c>.
+        /// </remarks>
         public void Dispose()
```

Comment:
The idea is that implementing IDisposable  allows you to use `using() {}` syntactic sugar to make sure the call is cancelled in case of abnormal code flow.  The Cancel() method wouldn't be useful as cancellation in normally done by a CancellationToken.

I was specifically asked to add the clarification in the API review.
(* Suggest stronger documentation of "you don't need to dispose of this normally". *) That is important as in .NET there are disposable objects that you are supposed to always dispose (like FileStream etc)  and disposable objects that you aren't expected to dispose (like Task).